code/modules/projectiles/guns/energy/laser_musket.dm

```diff
@@ -0,0 +1,32 @@
+/obj/item/gun/energy/laser/musket
+	name = "laser musket"
+	desc = "A hand-crafted laser weapon, it has a hand crank on the side to charge it up."
+	icon_state = "musket"
+	inhand_icon_state = "musket"
+	worn_icon_state = "las_musket"
+	ammo_type = list(/obj/item/ammo_casing/energy/laser/musket)
+	slot_flags = ITEM_SLOT_BACK
+	obj_flags = UNIQUE_RENAME
+	var/is_charging = FALSE
+	can_bayonet = TRUE
+	knife_x_offset = 22
+	knife_y_offset = 11
```

Comment:
```suggestion
	can_bayonet = TRUE
	knife_x_offset = 22
	knife_y_offset = 11
	/// Is the gun currently being charged?
	var/is_charging = FALSE
```
I've noticed that it's usually best to have new variables under the preexisting ones.

Reply:
Will do, appreciated!